src/mlpack/methods/ann/activation_functions/tanh_exponential_function.hpp

```diff
@@ -0,0 +1,83 @@
+/**
+ * @file methods/ann/activation_functions/tanh_exponential_function.hpp
+ * @author Mayank Raj
+ *
+ * Definition and implementation of the Tanh exponential  function.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_TANH_EXPONENTIAL_FUNCTION_HPP
+#define MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_TANH_EXPONENTIAL_FUNCTION_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * The TanhExp function, defined by
```

Comment:
Can you reference the paper here? https://github.com/mlpack/mlpack/blob/1b684afef5548c6795dc9ca3cb5f04eade23a14e/src/mlpack/methods/ann/activation_functions/elliot_function.hpp#L8-L17 is an example of how the format looks like.